.github/pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+<!--  Thanks for sending a pull request!  Here are some tips for you -->
+
+**What this PR does / why we need it**:
+
+**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
+
+**Special notes for your reviewer**:
+
+**Checklist:**
```

Comment:
this file as well doesn't match

Reply:
I am not sure in other repos there was a line `**PLEASE REMOVE THIS TEMPLATE BEFORE SUBMITTING**`

Reply:
About checklist I'm not sure if we need.
- [ ] Fixes #<issue number> : already mentioned at https://github.com/openebs/istgt/pull/323/files/c9986a1c96efeb0244164affbb0134160d0cb2b9#diff-6a3371457528722a734f3c51d9238c13R95.
- [ ] Labelled this PR & related issue with `documentation` tag : Requires permission
- [ ] PR messages has document related information : not required since maintainer/reviewer can ask if required
- [ ] Labelled this PR & related issue with `breaking-changes` tag : Requires permission
- [ ] PR messages has breaking changes related information : maintainer/reviewer can only decide
- [ ] Labelled this PR & related issue with `requires-upgrade` tag : requires permission
- [ ] PR messages has upgrade related information : maintainer/reviewer can only decide
- [ ] Commit has unit tests : maintainer/reviewer can go through the commit and if required same can be asked from contributor.
- [ ] Commit has integration tests : maintainer/reviewer can go through the commit and if required same can be asked from contributor.

Reply:
Oh I think it is for PR